src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -130,7 +141,9 @@ protected void runTool( final JavaSparkContext ctx ) {
 
         // write the output file
         final SAMFileHeader cleanHeader = new SAMFileHeader(header.getSequenceDictionary());
-        AlignedAssemblyOrExcuse.writeSAMFile(outputSAM, cleanHeader, alignedAssemblyOrExcuseList);
+        cleanHeader.setSortOrder(outputSortOrder);
```

Comment:
I have changed writerSAMFile get the order from the input header that also holds that information. I think make more sense to ask the user to provide a header whose content is according the intended order.